src/main/java/teammates/ui/webapi/RejectAccountRequestAction.java

```diff
@@ -0,0 +1,59 @@
+package teammates.ui.webapi;
+
+import java.util.UUID;
+
+import teammates.common.datatransfer.AccountRequestStatus;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Const;
+import teammates.common.util.EmailWrapper;
+import teammates.storage.sqlentity.AccountRequest;
+import teammates.ui.output.AccountRequestData;
+import teammates.ui.request.AccountRequestRejectionRequest;
+import teammates.ui.request.InvalidHttpRequestBodyException;
+
+/**
+ * Rejects an account request.
+ */
+public class RejectAccountRequestAction extends AdminOnlyAction {
+
+    @Override
+    public JsonResult execute() throws InvalidOperationException, InvalidHttpRequestBodyException {
+        String id = getNonNullRequestParamValue(Const.ParamsNames.ACCOUNT_REQUEST_ID);
+        UUID accountRequestId;
+
+        try {
+            accountRequestId = UUID.fromString(id);
+        } catch (IllegalArgumentException e) {
+            throw new InvalidHttpParameterException(e.getMessage(), e);
+        }
```

Comment:
I think there is already a method for this, `getUuidRequestParamValue`. It would probably be better to use that.

Reply:
@xenosf let's submit a separate PR to address this